model/riscv_sys_regs.sail

```diff
@@ -187,10 +188,10 @@ function haveNExt()    -> bool = misa.N() == 0b1
 /* see below for F and D (and Z*inx counterparts) extension tests */
 
 /* BitManip extension support. */
-function haveZba()  -> bool = true
-function haveZbb()  -> bool = true
+function haveZba()  -> bool = haveBExt()
```

Comment:
These should all remain true

Reply:
right. That would be backward compatible change without the isa string parser. Updated.